cmd/lncli/commands.go

```diff
@@ -373,6 +373,22 @@ func sendCoins(ctx *cli.Context) error {
 			"sweep all coins out of the wallet")
 	}
 
+	client, cleanUp := getClient(ctx)
+	defer cleanUp()
+	minConfs := int32(ctx.Uint64("min_confs"))
+
+	// In case that the user has specified the sweepall flag, we'll
+	// calculate the amount to send based on the current wallet balance.
+	if ctx.Bool("sweepall") {
+		balanceResponse, _ := client.WalletBalance(
+			ctxc, &lnrpc.WalletBalanceRequest{
+				Account:  ctx.String("account"),
+				MinConfs: minConfs,
```

Comment:
Ok, it makes sense to query the available balance with `minConfs` if we want to sweep all funds with at least `minConfs`.